sakura_core/cmd/CViewCommander_File.cpp

```diff
@@ -404,7 +404,7 @@ void CViewCommander::Command_PLSQL_COMPILE_ON_SQLPLUS( void )
 	if( GetDocument()->m_cDocFile.GetFilePathClass().IsValidPath() ){
 		/* ファイルパスに空白が含まれている場合はダブルクォーテーションで囲む */
 		//	2003.10.20 MIK コード簡略化
-		if( wcschr( GetDocument()->m_cDocFile.GetFilePath(), TCODE::SPACE ) ? TRUE : FALSE ){
+		if( wcschr( GetDocument()->m_cDocFile.GetFilePath(), WCODE::SPACE ) ? TRUE : FALSE ){
```

Comment:
これも指摘じゃないですが・・・。
元を書いた人がC言語あまり得意じゃなかったんですかね。

```suggestion
		if( wcschr( GetDocument()->m_cDocFile.GetFilePath(), WCODE::SPACE ) ){
```

これ、普通は Windows API を使って書く処理です。
https://docs.microsoft.com/en-us/windows/win32/api/shlwapi/nf-shlwapi-pathquotespacesw

このコードを実行させるためには Oracle Client とかインストールせにゃならん？（やらない、やらない。